gems/sorbet-runtime/lib/types/props/decorator.rb

```diff
@@ -449,8 +449,11 @@ def prop_defined(name, cls, rules={})
 
     @class.send(:define_method, redacted_method) do
       value = self.public_send(prop_name)
-      Chalk::Tools::RedactionUtils.redact_with_directive(
-        value, redaction)
+      handler = T::Configuration.redaction_handler
+      if !handler
+        raise RuntimeError, "No redaction handler configured"
```

Comment:
```suggestion
        raise "Using `redaction:` on a prop requires specifying `T::Configuration.redaction_handler`"
```

`RuntimeError` is implied